### DISCUSSION TITLE
`vim.uv` is nil when inside a distrobox with custom home directory

### DISCUSSION THREAD

**Discussion by J-Kappes** [NvChad/NvChad#3048] [Q&A] [answered]
I'm using VanillaOS's apx wrapper around distrobox.

The NVChad config doesn't get applied and throws the following error:
```
E5113: Error while calling lua chunk: /home/rj/Projekte/.config/nvim/init.lua:7: attempt to index field 'uv' (a nil value)
```
What is `vim.uv` and why would it break in a distrobox with a custom home directory?

**Comment by siduck** [comment, marked as answer]
you need v0.10 at least
